ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Questions
+about: Questions about project
+title: ''
+labels: question, triage
+assignees: ''
+---
+
+**Delete this: We would like to use GitHub for bug reports and feature requests only however if you are unable to get support in: [Neo Discord](https://discord.com/invite/R8v48YA) server or in our [offical website](https://fs.neo.org/), feel encouraged to create an issue here on GitHub.**
```

Comment:
Make it a comment (like in other templates) to minimize accidental posting of this exact content.

And maybe it'd be more like

> Before posting your question, try looking for an answer on [NeoFS website](), search for similar issues here or ask on [Neo Discord](). If nothing helps, submit this issue.

Maybe add some sections like

> ## Problem
> Describe what you're trying to accomplish and what's your problem...
>
> ## Context
> Describe your environment, OS, limitations, etc.

(needs to be written better of course)